htdocs/themes/xbootstrap/modules/publisher/blocks/publisher_items_recent.tpl

```diff
@@ -21,5 +21,5 @@
 </table>
 
 <div style="text-align:right; padding: 5px;">
-    <a class="btn btn-primary btn-xs" href="<{$publisher_url}>"><{$block.lang_visitItem}></a>
+    <a class="btn btn-primary btn-xs" href="<{$publisher_url|default:false}>"><{$block.lang_visitItem}></a>
 </div>
```

Comment:
Should do a test on `$publisher_url` being empty prior to display of this `<div> and skip display if it's empty.
```suggestion
<{if $publisher_url|default:false}>
<div style="text-align:right; padding: 5px;">
    <a class="btn btn-primary btn-xs" href="<{$publisher_url}>"><{$block.lang_visitItem}></a>
</div>
<{/if}>
```